src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -44,8 +46,8 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             optionsBuilder.UseOracle(connectionString, options => options.UseOracleSQLCompatibility("11"));
 
             // can optionally Log any SQL that is ran by uncommenting:
-            optionsBuilder.UseLoggerFactory(MyLoggerFactory);
-            optionsBuilder.EnableSensitiveDataLogging(true);
+            // optionsBuilder.UseLoggerFactory(MyLoggerFactory);
```

Comment:
why comment out and not delete?

Reply:
can be useful to uncomment if you want to see debug sql